src/main/java/org/opentripplanner/profile/RaptorWorkerData.java

```diff
@@ -0,0 +1,233 @@
+package org.opentripplanner.profile;
+
+import com.beust.jcommander.internal.Lists;
+import com.google.common.collect.Iterables;
+
+import gnu.trove.list.TIntList;
+import gnu.trove.list.array.TIntArrayList;
+import gnu.trove.map.TIntIntMap;
+import gnu.trove.map.TIntObjectMap;
+import gnu.trove.map.TObjectIntMap;
+import gnu.trove.map.hash.TIntIntHashMap;
+import gnu.trove.map.hash.TIntObjectHashMap;
+import gnu.trove.map.hash.TObjectIntHashMap;
+
+import org.onebusaway.gtfs.model.Stop;
+import org.opentripplanner.analyst.SampleSet;
+import org.opentripplanner.routing.edgetype.SimpleTransfer;
+import org.opentripplanner.routing.edgetype.TripPattern;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.graph.Vertex;
+import org.opentripplanner.routing.vertextype.TransitStop;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class RaptorWorkerData implements Serializable {
```

Comment:
Likewise, this needs Javadoc explaining what it's for, and I'll add some after the merge.